test/functional/assertion-helper.js

```diff
@@ -0,0 +1,8 @@
+var expect = require('chai').expect;
+
+
+exports.errorInEachBrowserContains = function errorInEachBrowserContains (testError, message, errorIndex) {

```

Comment:
you can use an unnamed function here because it's not used in the file
```js
exports.errorInEachBrowserContains = function (testError, message, errorIndex) {
```